optimize/self-managed/optimize-deployment/migration-update/2.6-to-2.7.md

```diff
@@ -23,19 +23,19 @@ With this Optimize version, there are also changes in the supported versions of
 ### Elasticsearch
 
 Optimize now requires at least Elasticsearch `6.4.0`.
-See the [Supported Environments](./../../../reference/supported-environments.md) sections for the full range of supported versions.
+See the [Supported Environments]($docs$/reference/supported-environments) sections for the full range of supported versions.
```

Comment:
...and again, some of these that moved _into_ `optimize/` needed to be updated to cross-link to the main `docs/` instance.